add_new_dc_test.py

```diff
@@ -2,7 +2,7 @@
 from typing import Tuple, List
 
 from cassandra import ConsistencyLevel
-from cassandra.query import SimpleStatement  # pylint: disable=no-name-in-module
```

Comment:
why removing these disable statements?

Reply:
since they are pylint related, and I suggesting at moving to a different tool

Reply:
but some checks looks like valid checks. Now they will be missed?

Reply:
those comments doesn't define a check, they override it
are you asking if we have the equivalent in ruff, I don't think so, since it's doesn't import the code some as pylint can.

Reply:
yes, we override because pylint check would cause to fail - now ruff will omit some checks (this one is just example), but we don't know if they don't support it in the future and cause fail then.

Reply:
Then once we enable certain checks of new features in ruff we will add what is need, leaving the pylint comments would be much help (since ruff doesn't read them)

Reply:
I decided not to remove the pylint comment for no, I'm leaving them as is